### ISSUE TITLE
Trailing question mark in title changes howpublished from online to Online

### ISSUE THREAD

**Issue by Witiko** [michal-h21/biblatex-iso690#95]
Here is a minimal working example:

``` tex
\documentclass{article}
\usepackage[czech]{babel}
\usepackage{expl3,csquotes}
\usepackage[style=iso-authoryear]{biblatex}
\begin{filecontents}{mwe.bib}

@online{with-question-mark,
    author    = {SÚKL},
    title     = {eRecept: Co je to identifikátor eReceptu?},
    url       = {https://www.epreskripce.cz/co-je-identifikator-ereceptu},
    urldate   = {2021-09-29},
    location  = {Brno},
    langid    = {czech}
}

@online{without-question-mark,
    author    = {SÚKL},
    title     = {eRecept: Co je to identifikátor eReceptu},
    url       = {https://www.epreskripce.cz/co-je-identifikator-ereceptu},
    urldate   = {2021-09-29},
    location  = {Brno},
    langid    = {czech}
}

\end{filecontents}
\addbibresource{mwe.bib}
\begin{document}
\nocite{*}
\printbibliography
\end{document}

```

I typeset the example with [a current TeX Live Docker image](https://hub.docker.com/layers/texlive/texlive/latest/images/sha256-68c35d645848342722316bf505b5d4ed528107f62cb60bbd324e1c5a836cd529?context=explore):

```
docker run --rm -v "$PWD":/workdir -w /workdir texlive/texlive@sha256:68c35d645848342722316bf505b5d4ed528107f62cb60bbd324e1c5a836cd529 latexmk -pdf mwe.tex
```

I receive the following output:

![mwe](https://user-images.githubusercontent.com/603082/141849825-60fb9ed3-f551-4367-a605-d88af377fa99.png)

Apparently, adding a question mark at the end of the title changes howpublished from *online* to *Online*. As a workaround, I can force the correct text by specifying `howpublished = {online}`.

**Comment by michal-h21** [comment]
@Witiko thanks for the report. I've updated the macro that prints `[online]` string to ignore the punctuation tracker, so it should always print the string in lowercase.

**Comment by DavidLuptak** [comment]
- [x] Implemented in 4013b705cdbab8d1bdd0145cd1c0f3f726a7634b.

Closing the issue now.
